.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
this file maybe at your local only.

Reply:
Ahh yeah I'll remove it and add this to the `.gitignore` 👌